.travis.yml

```diff
@@ -19,9 +19,12 @@ matrix:
     script: bundle exec rake beaker
     services: docker
     sudo: required
+  - rvm: 2.3.1
+    bundler_args: --without system_tests
+    env: PUPPET_GEM_VERSION="~> 4.0"
   - rvm: 2.1.9
     bundler_args: --without system_tests
-    env: PUPPET_GEM_VERSION="~> 4.0" STRICT_VARIABLES="yes"
```

Comment:
puppetlabs_spec_helper 1.2 defaults to strict variable testing for puppet 4. (it's still disableable)